src/commands/deploy.ts

```diff
@@ -49,11 +50,13 @@ $ bump deploy FILE --dry-run --doc <doc_slug> --token <your_doc_token>
   */
   async run(): Promise<void> {
     const { args, flags } = this.parse(Deploy);
-    const [definition, references] = await this.prepareDefinition(args.FILE);
+    const api = await API.load(args.FILE);
```

Comment:
Again, a question for understanding this typescript PR:

Before the refacto, this function `prepareDefinition` is moved from shared script `command.ts` to dedicated class `Diff`, that is called by this other class `API`, defined in `src/definition.ts`, right ?

Reply:
The `command.ts`'s `prepareDefinition` is moved inside the `API` object (defined in `src/definition.ts`) yes.

Thus the function call is now replaced by:
- `api = API.load(file)` : Load the definition file into the `API` object
- then extract the definition and references `api.extractDefinition()` (needed to call Bump.sh's API)

Reply:
Crystal clear, thanks